src/DynamoCore/Models/RunSettings.cs

```diff
@@ -106,6 +106,7 @@ public RunSettings(RunType runType, int period)
         internal void Reset()
         {
             RunEnabled = true;
+            RunType = RunType.Automatic;
```

Comment:
This line was initially removed on this PR because it was overriding the user's Default Setting. After some changes to the pr this should have been added back and that is what caused the test to fail